olive/model/utils/onnx_utils.py

```diff
@@ -67,3 +71,111 @@ def get_onnx_file_path(model_path: str, onnx_file_name: Optional[str] = None) ->
         )
     else:
         raise ValueError(f"No .onnx file found in the model folder {model_path}.")
+
+
+def check_and_normalize_provider_args(
+    providers: Sequence[Union[str, Tuple[str, Dict[Any, Any]]]],
+    provider_options: Sequence[Dict[Any, Any]],
+    available_provider_names: Sequence[str],
+):
+    """Validate the 'providers' and 'provider_options' arguments and returns a normalized version.
```

Comment:
Does `copy but not call from ort` because this function only exists after specific ort version?

Reply:
This function is not exposed. I copy here just avoid the signature changes breaking Olive in future.